header.php

```diff
@@ -53,9 +53,56 @@
 <body <?php body_class(); ?>>
 <!--[if lt IE 9]><p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p><![endif]-->
 
-  <section id="main-container">
+<section id="main-container">
 
-  <!-- start content -->
-  <header id="header">
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
-  </header>
+  <section id="menus" class="u-cf">
+
+    <header id="header" class="menu-column">
+      <nav class="menu-column-top">
+        *menu icon*
+      </nav>
```

Comment:
Are these `<nav>` the toggle buttons? I think it doesn't make any semantic sense to use nav here.

> Represents navigation for a document. The nav element is a section containing links to other documents or to parts within the current document.
Not all groups of links on a page need to be in a nav element — only groups of primary navigation links. In particular, it is common for footers to have a list of links to various key parts of a site, but the footer element is more appropriate in such cases.

I think these should be just divs.
`#menus` should be a `div` instead of header since it doesn't contain any kind of Page Title information(ex `h1`).
`.menu-column-content` should be the `nav`s